### DISCUSSION TITLE
Feature Request: Kiosk Mode or Hide Tab Menu Option

### DISCUSSION THREAD

**Discussion by scramb123** [jessebofill/DeckWebBrowser#7] [Ideas]
I would like to propose the introduction of a "Kiosk Mode" or an option to hide the tab menu, aiming to provide a cleaner user interface. This feature would be particularly beneficial for users like myself, who utilize the application as a quick and efficient way to control Home Assistant while engaged in gaming. 

A streamlined interface, free from unnecessary elements, would greatly enhance the ease of use and accessibility, making for a smoother and more convenient interaction during gaming sessions.

**Comment by jessebofill** [comment]
I’ll have to look into this. How would propose it would work? Would it just be something the user would enable/disable while in the browser page?

**Comment by jessebofill** [comment]
Actually I’m thinking it would be something you would have to set in the settings menu, and it would just allow single tab usage.

**Comment by jessebofill** [comment]
This feature has been implemented in 1.3.0 which just released on decky store a few days ago. You can have it enabled/ disabled by default as well as quickly toggle it on/off in the browser context menu itself. Also it still works with multiple tabs, the tab bar will just be hidden, and just still switch tabs with L/R buttons.
